engine/flink/util/src/main/scala/pl/touk/nussknacker/engine/flink/util/function/CoProcessFunctionInterceptor.scala

```diff
@@ -0,0 +1,49 @@
+package pl.touk.nussknacker.engine.flink.util.function
+
+import org.apache.flink.api.common.functions.RuntimeContext
+import org.apache.flink.configuration.Configuration
+import org.apache.flink.streaming.api.functions.co.CoProcessFunction
+import org.apache.flink.util.Collector
+
+abstract class CoProcessFunctionInterceptor[IN1, IN2, OUT](underlying: CoProcessFunction[IN1, IN2, OUT]) extends CoProcessFunction[IN1, IN2, OUT] {
```

Comment:
Is it meant to be used for sth else than tests? If not, let's add at least comment?

Reply:
It can be used anywhere - also for some extra logs / notifications in production code. But indeed currently it is used only in tests. I can add some docs about possible usages of this class.

Reply:
ok, thanks